### ISSUE TITLE
calendar DELETE now causes a 500 internal server error

### ISSUE THREAD

**Issue by tobixen** [jelmer/xandikos#210]
Master branch ... I'll tro to do a bisect again

```
  File "/home/tobias/caldav/tests/test_caldav.py", line 2196, in xandikos_handler
    return await mainapp.aiohttp_handler(request, "/")
  File "/usr/lib/python3.10/site-packages/xandikos-0.2.8-py3.10.egg/xandikos/webdav.py", line 2152, in aiohttp_handler
    response = await self._handle_request(request, environ)
  File "/usr/lib/python3.10/site-packages/xandikos-0.2.8-py3.10.egg/xandikos/webdav.py", line 2108, in _handle_request
    return await do.handle(request, environ, self)
  File "/usr/lib/python3.10/site-packages/xandikos-0.2.8-py3.10.egg/xandikos/webdav.py", line 1645, in handle
    pr.delete_member(item_name, current_etag)
  File "/usr/lib/python3.10/site-packages/xandikos-0.2.8-py3.10.egg/xandikos/web.py", line 384, in delete_member
    self.store.delete_one(name, etag=extract_strong_etag(etag))
  File "/usr/lib/python3.10/site-packages/xandikos-0.2.8-py3.10.egg/xandikos/store/git.py", line 742, in delete_one
    with open(p, "rb") as f:
IsADirectoryError: [Errno 21] Is a directory: '/tmp/tmpnjdl6lkc/sometestuser/calendars/pythoncaldav-test2'
```

**Comment by tobixen** [comment]
Hm, I was probably running some exotic version of xandikos and not the latest master.  I can't reproduce it now.  I'll just close this one ...

**Comment by jelmer** [comment]
Yeah, this should have been addressed by 91dcb30e9de72037d81c159751e18cce8010f1e6
